tutorials/math/random_number_generation.rst

```diff
@@ -196,6 +326,41 @@ We can apply similar logic from arrays to dictionaries as well::
         # The same metal may be selected multiple times in succession.
         return random_metal
 
+ .. code-tab:: csharp
+
+    struct MetalInfo
+    {
+        public MetalInfo(int quantity, int price)
+        {
+            this.quantity = quantity;
+            this.price = price;
+        }
+        public int quantity;
+        public int price;
+    }
+
+    enum MetalType
+    {
+        copper, 
+        silver, 
+        gold,
+    }
+    Godot.Collections.Dictionary<MetalType, MetalInfo> metals = new Godot.Collections.Dictionary<MetalType, MetalInfo>
+    {
+        { MetalType.copper, new MetalInfo(50, 50) }, 
+        { MetalType.silver, new MetalInfo(20, 150) }, 
+        { MetalType.gold, new MetalInfo(3, 500) }
+    };
+
+    public override void _Ready()
+    {
+        GD.Randomize();
+        for (int i = 0; i < 20; i++)
+        {
+            GD.Print(get_meta());
```

Comment:
This method call isn't PascalCase and isn't defined anywhere anyway.

Reply:
I said in the OP, I don't know how to make the function that method is for